Sources/CodeEditSourceEditor/CodeEditSourceEditor/CodeEditSourceEditor.swift

```diff
@@ -72,7 +75,9 @@ public struct CodeEditSourceEditor: NSViewControllerRepresentable {
         useSystemCursor: Bool = true,
         undoManager: CEUndoManager? = nil,
         coordinators: [any TextViewCoordinator] = [],
-        showMinimap: Bool
+        showMinimap: Bool,
+        reformatAtColumn: Int,
```

Comment:
Could we make this an optional `Int?` instead of two parameters? That way the state is in a single variable (enabled: `some(Int)`, disabled: `nil`)

Reply:
If you'd prefer hiding be a little more explicit I'd suggest an enum instead.
```swift
enum ReformattingGuidePosition {
    case hidden
    case visible(column: Int)
}
```

But imo the optional int works just as well.

Reply:
Xcode has two separate settings here.

<img width="510" alt="image" src="https://github.com/user-attachments/assets/acd27c21-c663-4c37-aba9-3f65de986d5c" />

I think this is because this column number isn't exclusively for the guide, but is also used for general reformatting.

Reply:
Oh good point, we'll keep it as two params then.